messages/cache_setting/create.go

```diff
@@ -17,7 +17,7 @@ const (
 	CreateFlagCdnCacheSettings           = "Configures the amount of time Azion's Edge Applications take to cache the content. It can either Honor Origin Cache Headers or Override Cache Settings"
 	CreateFlagCachingForOptions          = "Whether caching for options is active or not"
```

Comment:
I don't wanna be a pain in the ass, but this should have the enabled as well right? 😬

Reply:
Thank you for your comments :smile: